sys/linux/fs_fuse.txt

```diff
@@ -142,8 +169,12 @@ fuse_write_out {
 	padding	const[0, int32]
 }
 
+fuse_read_out {
+	content	string
+}
+
 fuse_open_out {
-	fh		const[0, int64]
+	fh		int64
```

Comment:
Why?

Reply:
From my understanding, fh will be used by the kernel to refer to the resource being opened. It is supplied by the FUSE daemon and there are no restriction on its specific value. Therefore, in my opinion, it makes more sees not to hardcode it to 0. What do you think?

Reply:
Does kernel use it in any way? From briefly skimming through the kenrel code, it looks like an opaque handle for userspace.

Reply:
I haven't checked, but it probably doesn't. Still, this might help catch bugs that can be introduced in the future.

Reply:
This is exactly the opposite argument you make with the pseudo-syscalls ;)

Reply:
Also semantics of this field can't change in future, it's part of stable kernel API with lots of existing programs passing arbitrary data in this field already.

Reply:
I see your point, I have no problems dropping the change. Thanks for pointing that out!